wolfcrypt/src/ecc.c

```diff
@@ -7975,7 +7975,12 @@ int wc_ecc_import_private_key_ex(const byte* priv, word32 privSz,
     }
 #elif defined(WOLFSSL_SILABS_SE_ACCEL)
     if (ret == MP_OKAY) {
-      ret = silabs_ecc_import_private(key, privSz);
+        if (pub) {
+            ret = silabs_ecc_import(key, privSz);
```

Comment:
Should this be `pubSz`?

Reply:
Both are key size now as the calls expect.